rsocket-core/src/main/java/io/rsocket/Plugins.java

```diff
@@ -1,49 +0,0 @@
-package io.rsocket;
-
-import io.rsocket.fragmentation.FragmentationDuplexConnection;
-import java.util.function.BiFunction;
-import java.util.function.Function;
-import reactor.core.publisher.Mono;
-
-/** */
-public class Plugins {
-
-  public interface DuplexConnectionInterceptor
-      extends BiFunction<DuplexConnectionInterceptor.Type, DuplexConnection, DuplexConnection> {
-    enum Type {
-      STREAM_ZERO,
-      CLIENT,
-      SERVER,
-      SOURCE
-    }
-  }
-
-  public interface RSocketInterceptor extends Function<RSocket, Mono<RSocket>> {}
-
-  public static final DuplexConnectionInterceptor NOOP_DUPLEX_CONNECTION_INTERCEPTOR =
-      (type, connection) -> connection;
-  private static final RSocketInterceptor NOOP_INTERCEPTOR = Mono::just;
-
-  public static volatile DuplexConnectionInterceptor DUPLEX_CONNECTION_INTERCEPTOR =
-      NOOP_DUPLEX_CONNECTION_INTERCEPTOR;
-  public static volatile RSocketInterceptor CLIENT_REACTIVE_SOCKET_INTERCEPTOR = NOOP_INTERCEPTOR;
-  public static volatile RSocketInterceptor SERVER_REACTIVE_SOCKET_INTERCEPTOR = NOOP_INTERCEPTOR;
-
-  private Plugins() {}
-
-  static {
-    if (Boolean.getBoolean("io.rsocket.fragmentation.enable")) {
-      int mtu = Integer.getInteger("io.rsocket.fragmentation.mtu", 1024);
```

Comment:
This is removed as its already supported in the Builder, so I can move this system property logic to become the default MTU.

If I leave it here, you get fragmentation twice if user overrides.